packages/wrangler/src/pages/upload.tsx

```diff
@@ -368,6 +368,16 @@ export const upload = async (
 	);
 };
 
+// Decode and check that the current JWT has not expired
+function isJwtExpired(token: string): boolean {
+	const decodedJwt = JSON.parse(
+		Buffer.from(token.split(".")[1], "base64").toString()
+	);
+	const dateNow = new Date().getTime() / 1000;
+
+	return decodedJwt.exp < dateNow;
```

Comment:
I know it's pretty unlikely, but this should be:

```suggestion
	return decodedJwt.exp <= dateNow;
```